Source/ablastr/utils/SignalHandling.cpp

```diff
@@ -191,7 +191,7 @@ SignalHandling::WaitSignals ()
     }
 
 #if defined(AMREX_USE_MPI)
-    BL_MPI_REQUIRE(MPI_Wait(&signal_mpi_ibcast_request, MPI_STATUS_IGNORE));
+    BL_MPI_REQUIRE(MPI_Wait(&signal_mpi_ibcast_request, MPI_STATUS_IGNORE)); // NOLINT(clang-analyzer-optin.mpi.MPI-Checker)
```

Comment:
This seems like a common pattern that could need a lot of silencing down the road. Would not add this test.

Reply:
It's the only check of the `clang-analyzer-*` group not enabled in AMReX. It could be OK to disable it also for WarpX.

Reply:
Done. The check has been disabled and `//NOLINT` has been removed.